File: result/translate/starchat/java2cpp/c++/PERMUTE_TWO_ARRAYS_SUM_EVERY_PAIR_GREATER_EQUAL_K.cpp
bool f_gold (vector<int> a, vector<int> b, int n, int k) {
        sort(a.rbegin(), a.rend());
        sort(b.begin(), b.end());
        for (int i = 0; i < n; i++)
            if (a[i] + b[i] < k)
                return false;
        return true;
    }

File: result/translate/transcoder/python2java/java/PERMUTE_TWO_ARRAYS_SUM_EVERY_PAIR_GREATER_EQUAL_K.java
public static boolean f_gold ( int [ ] a , int [ ] b , int n , int k ) {
  int [ ] a = new int [ n ] ;
  int [ ] b = new int [ n ] ;
  Arrays . sort ( a ) ;
  Arrays . sort ( b ) ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) {
    if ( ( a [ i ] + b [ i ] ) < k ) return false ;
  }
  return true ;
}